pkgs/desktops/gnome/core/gdm/default.nix

```diff
@@ -151,13 +159,7 @@ stdenv.mkDerivation rec {
     glib-compile-schemas "$out/share/glib-2.0/schemas"
   '';
 
-  # HACK: We want to install configuration files to $out/etc
-  # but GDM should read them from /etc on a NixOS system.
-  # With autotools, it was possible to override Make variables
-  # at install time but Meson does not support this
-  # so we need to convince it to install all files to a temporary
-  # location using DESTDIR and then move it to proper one in postInstall.
-  DESTDIR = "${placeholder "out"}/dest";
```

Comment:
Why move this?

Reply:
I thought it's easier to put this into let in instead of write finalAttrs everywhere.

Reply:
It does not use `finalAttrs` though?

Reply:
I need to use `${finalAttrs.DESTDIR}`. Should I use `$DESTDIR` instead?

Reply:
Ah, the `postInstall`. It should not really be interpolating the value in the first place, picking it up from the environment `$DESTDIR` should be sufficient).